urllib: flaky test with python 3.8

Flaky tests with Python 3.8

```
 =================================== FAILURES ===================================
__________ TestUrllibMetricsInstrumentation.test_metric_uninstrument ___________

self = <tests.test_metrics_instrumentation.TestUrllibMetricsInstrumentation testMethod=test_metric_uninstrument>

    @mark.skipif(
        python_implementation() == "PyPy", reason="Fails randomly in pypy"
    )
    def test_metric_uninstrument(self):
        with request.urlopen(self.URL):
            self.assertEqual(
                len(
                    (
                        self.memory_metrics_reader.get_metrics_data()
                        .resource_metrics[0]
                        .scope_metrics[0]
                        .metrics
                    )
                ),
                3,
            )
    
>           self.assertEqual(
                (
                    self.memory_metrics_reader.get_metrics_data()
                    .resource_metrics[0]
                    .scope_metrics[0]
                    .metrics[0]
                    .data.data_points[0]
                    .bucket_counts[1]
                ),
                1,
            )
E           AssertionError: 0 != 1

instrumentation/opentelemetry-instrumentation-urllib/tests/test_metrics_instrumentation.py:419: AssertionError

```

--- a/instrumentation/opentelemetry-instrumentation-urllib/tests/test_metrics_instrumentation.py
+++ b/instrumentation/opentelemetry-instrumentation-urllib/tests/test_metrics_instrumentation.py
@@ -414,16 +414,16 @@ class TestUrllibMetricsInstrumentation(TestBase):
             ) = metrics[:3]
 
             self.assertEqual(
-                client_duration.data.data_points[0].bucket_counts[1],
+                sum(client_duration.data.data_points[0].bucket_counts),
                 1,
             )
 
             self.assertEqual(
-                client_request_size.data.data_points[0].bucket_counts[0],
+                sum(client_request_size.data.data_points[0].bucket_counts),
                 1,
             )
             self.assertEqual(
-                client_response_size.data.data_points[0].bucket_counts[2],
+                sum(client_response_size.data.data_points[0].bucket_counts),
                 1,
             )
 
@@ -439,15 +439,15 @@ class TestUrllibMetricsInstrumentation(TestBase):
             ) = metrics[:3]
 
             self.assertEqual(
-                client_duration.data.data_points[0].bucket_counts[1],
+                sum(client_duration.data.data_points[0].bucket_counts),
                 2,
             )
             self.assertEqual(
-                client_request_size.data.data_points[0].bucket_counts[0],
+                sum(client_request_size.data.data_points[0].bucket_counts),
                 2,
             )
             self.assertEqual(
-                client_response_size.data.data_points[0].bucket_counts[2],
+                sum(client_response_size.data.data_points[0].bucket_counts),
                 2,
             )
 
@@ -467,15 +467,15 @@ class TestUrllibMetricsInstrumentation(TestBase):
             self.assertEqual(len(metrics), 3)
 
             self.assertEqual(
-                client_duration.data.data_points[0].bucket_counts[1],
+                sum(client_duration.data.data_points[0].bucket_counts),
                 2,
             )
             self.assertEqual(
-                client_request_size.data.data_points[0].bucket_counts[0],
+                sum(client_request_size.data.data_points[0].bucket_counts),
                 2,
             )
 
             self.assertEqual(
-                client_response_size.data.data_points[0].bucket_counts[2],
+                sum(client_response_size.data.data_points[0].bucket_counts),
                 2,
             )

